common/djangoapps/pipeline_mako/templates/static_content.html

```diff
@@ -132,3 +133,11 @@
 <%def name="show_language_selector()"><%
     return settings.FEATURES.get('SHOW_LANGUAGE_SELECTOR', False)
 %></%def>
+
+<%def name="is_user_authenticated()"><%
```

Comment:
Ah, yeah, that was something I suggested -- I think I maybe have gotten overzealous with our use of static -- it seems generally-acceptable to reference the user object directly in templates, so I am okay reverting back to using it.